contracts/extensions/bridge-operator-governance/BridgeAdminOperator.sol

```diff
@@ -0,0 +1,423 @@
+    return _getBridgeOperatorSet().length();
+  }
+
+  /**
+   * @inheritdoc IBridgeAdminOperator
+   */
+  function isBridgeOperator(address addr) external view returns (bool) {
+    return _getBridgeOperatorSet().contains(addr);
+  }
+
+  /**
+   * @inheritdoc IBridgeAdminOperator
+   */
+  function getBridgeOperators() public view returns (address[] memory) {
+    return _getBridgeOperatorSet().values();
+  }
+
+  /**
+   * @inheritdoc IBridgeAdminOperator
+   */
+  function getGovernors() external view returns (address[] memory) {
+    return _getGovernorsSet().values();
+  }
+
+  /**
+   * @inheritdoc IBridgeAdminOperator
+   */
+  function getBridgeOperatorOf(address[] calldata governors) external view returns (address[] memory bridgeOperators_) {
+    uint256 length = governors.length;
+    bridgeOperators_ = new address[](length);
+
+    mapping(address => BridgeOperatorInfo) storage gorvernorToBridgeOperator = _getGovernorToBridgeOperatorInfo();
+    for (uint256 i; i < length; ) {
+      bridgeOperators_[i] = gorvernorToBridgeOperator[governors[i]].addr;
+      unchecked {
+        ++i;
+      }
+    }
+  }
+
+  /**
+   * @inheritdoc IQuorum
+   */
+  function minimumVoteWeight() public view virtual returns (uint256) {
+    return (_num * _totalWeight + _denom - 1) / _denom;
+  }
+
+  function _addBridgeOperators(
+    uint256[] memory voteWeights,
+    address[] memory governors,
+    address[] memory brigdeOperators
+  ) internal returns (bool[] memory addeds) {
+    uint256 length = brigdeOperators.length;
+    addeds = new bool[](length);
+
+    EnumerableSet.AddressSet storage governorSet = _getGovernorsSet();
+    mapping(address => address) storage _governorOf = _getGovernorOf();
+    EnumerableSet.AddressSet storage bridgeOperatorSet = _getBridgeOperatorSet();
+    mapping(address => BridgeOperatorInfo) storage governorToBridgeOperatorInfo = _getGovernorToBridgeOperatorInfo();
+
+    address governor;
+    uint96 voteWeight;
+    address bridgeOperator;
+    uint256 accumulateWeight;
+    BridgeOperatorInfo memory bridgeOperatorInfo;
+    for (uint256 i; i < length; ) {
+      governor = governors[i];
+      bridgeOperator = brigdeOperators[i];
+      voteWeight = voteWeights[i].toUint96();
+
+      _checkNonZeroAddress(governor);
+      _checkNonZeroAddress(bridgeOperator);
+      _sanityCheck(voteWeight, governor, bridgeOperator);
+
+      addeds[i] = bridgeOperatorSet.add(bridgeOperator);
+
+      if (addeds[i]) {
+        governorSet.add(governor);
+
+        accumulateWeight += voteWeight;
+
+        _governorOf[bridgeOperator] = governor;
+
+        bridgeOperatorInfo.addr = bridgeOperator;
+        bridgeOperatorInfo.voteWeight = voteWeight;
+        governorToBridgeOperatorInfo[governor] = bridgeOperatorInfo;
+      }
+
+      unchecked {
+        ++i;
+      }
+    }
+
+    _totalWeight += accumulateWeight;
+
+    emit BridgeOperatorSetModified(msg.sender, BridgeAction.Add);
```

Comment:
Looks like this event misses some data related to the bridge operator set?

Reply:
fixed.